ui/perfherder/helpers.js

```diff
@@ -22,7 +22,9 @@ import {
 } from './constants';
 
 export const displayNumber = input =>
-  Number.isNaN(input) ? 'N/A' : Number(input).toFixed(2);
+  Number.isNaN(input)
+    ? 'N/A'
+    : (Math.round(Number(input) * 1e2) / 1e2).toLocaleString();
```

Comment:
This looks like it works well but you also use `Intl.NumberFormat` without needing to round and divide the number (it's used elsewhere in the codebase).